docs/docs/PythonUserGuide/install-from-pip.md

```diff
@@ -1,8 +1,8 @@
 ## **NOTES**
 
-- Pip install support __mac__ and __linux__ platform but only __Spark1.6.x__ for now.
-- Pip install only support run in __local__. Might support cluster mode in the future.
-- We've tested this package with __python 2.7__ and __python 3.5__
+- Pip install supports __Mac__ and __Linux__ platforms but only __Spark1.6.x__ for now.
+- Pip install only supports __local__ mode. Might support cluster mode in the future.
```

Comment:
Add a link to "install without pip" for people to use it in cluster mode.

Reply:
Updated.